doc/source/download.rst

```diff
@@ -84,21 +84,67 @@ legacy results.
 | md5  | e21be3e7d6a07ee439faf61e769667e4 |
 +------+----------------------------------+
 
-Drivable Maps
+Drivable Area
 ~~~~~~~~~~~~~~
 
-Segmentation maps of Drivable areas.
+Masks, colormaps and original json files for drivable area.
+The mask format is explained at: `Semantic Segmentation Format <https://doc.bdd100k.com/format.html#semantic-segmentation-format>`_
```

Comment:
We may change the section title. So better to use ref here. Same with the other HTTPS links.